ParallaxGen/src/main.cpp

```diff
@@ -136,10 +143,10 @@ void mainRunner(ParallaxGenCLIArgs &Args, const filesystem::path &ExePath) {
                Args.OutputDir.wstring());
 
   // Create bethesda game type object
-  BethesdaGame::GameType BGType = getGameTypeMap().at(Args.GameType);
+  BethesdaGame::GameType BGType = getGameTypeMap().at(Args.GameType); // NOLINT
```

Comment:
what's the nolint for here?

Reply:
checker thinks the variable is not initialized...